crates/rrg/Cargo.toml

```diff
@@ -130,3 +130,12 @@ features = [
     "Win32_Foundation",
     "Win32_Storage_FileSystem",
 ]
+
+[dependencies.ed25519-dalek]
+version = "2.1.1"
+features = [
+    "rand_core",
+]
```

Comment:
If I understand correctly, we only need `rand_core` for testing, so it should be only specified in `dev-dependencies`.

Reply:
Yes exactly! Updated.